### PR TITLE
🚸(app-desk) cannot select user or email already selected

### PR DESCRIPTION
**Pull request by AntoLC**
## Purpose

Cannot select user or email already selected.

Lot of flakiness was appearing, we remove the tests e2e with Firefox.

## Proposal

- [x] Add the teamid to the useUsers query, to not get the users that are already in the team
- [x] Add a check to not select a user or email that is already selected
- [x] Remove Firefox from tests e2e

### CODE REVIEW

File: src/frontend/apps/desk/src/features/addMembers/components/SearchMembers.tsx [hunk 1]
```diff
@@ -41,14 +43,24 @@ export const SearchMembers = ({
...
+    let users: OptionsSelect = optionsFiltered.map((user) => ({
       value: user,
       label: user.name || user.email,
       type: OptionType.NEW_MEMBER,
     }));
 
     if (userQuery && isValidEmail(userQuery)) {
-      const isFound = !!options.find((user) => user.email === userQuery);
+      let isFound = !!optionsFiltered.find((user) => user.email === userQuery);
+      isFound = !!selectedMembers.find(
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/features/addMembers/components/SearchMembers.tsx:61]: I feel there is a redundant initializer

**Review by lebaudantoine — COMMENTED** [review]
I might have miss something, lmk if my comment is irrelevant

File: .github/workflows/people.yml [hunk 2]
```diff
@@ -105,7 +105,7 @@ jobs:
   test-e2e:
     runs-on: ubuntu-latest
     needs: build-mails
-    timeout-minutes: 10
+    timeout-minutes: 20
```
[the remark this answers is not in the document]
Comment (AntoLC) [reply .github/workflows/people.yml]: Yes you'r right, we removed already Firefox, it should be enough.

File: src/frontend/apps/desk/src/features/addMembers/components/SearchMembers.tsx (hunk 1 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/addMembers/components/SearchMembers.tsx]: You'r right again, it didn't make any sense.
[0757799](https://github.com/numerique-gouv/people/pull/165/commits/0757799d4e21a0c66b8fbb0c01db59f7ddd70c45)

File: .github/workflows/people.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (AntoLC) [reply .github/workflows/people.yml]: [360d361](https://github.com/numerique-gouv/people/pull/165/commits/360d361dd3256be19816f9948fe53455d7400034)

**Review by lebaudantoine — COMMENTED** [review]
LGTM let's wait for @sdemagny to merge her PR

**Review by lebaudantoine — APPROVED** [review]

**Pull request merged**